cmd/lambda/approvals/handler.go

```diff
@@ -63,6 +63,7 @@ func buildHandler() (*Lambda, error) {
 	api, err := api.New(ctx, api.Opts{
 		Log:                 log,
 		DynamoTable:         cfg.DynamoTable,
+		KmsKey:              cfg.KMSKey,
```

Comment:
standardise capitalisation, you will need to apply this everywhere it is used
```suggestion
		KMSKeyARN:              cfg.KMSKeyARN,
```